tsconfig.json

```diff
@@ -43,6 +43,7 @@
     "src/composables/window.js",
     "src/constants/media.js",
     "src/constants/screens.js",
+    "src/constants/filters",
```

Comment:
You could probably add the filters store here too with all the type updates you made to it!